contracts/interfaces/IGitConsensus.sol

```diff
@@ -163,30 +163,20 @@ interface IGitConsensus is IGitConsensusErrors, IGitConsensusEvents, IGitConsens
         uint256[] calldata values
     ) external returns (bytes20 tagHash);
 
-    /// @notice Get the owner address of a git commit.
-    /// @param commitHash The hash of the git commit. If the commit that generated this hash does
+    /// @param gitHash The hash of the git commit. If the commit that generated this hash does
     ///     not exist, it means that no corresponding commit was ever added to the contract.
```

Comment:
/// not exist, it means that no corresponding commit or tag was ever added to the contract.